### DISCUSSION TITLE
Add docker support for the benchmark tool

### DISCUSSION THREAD

**Discussion by connorjburton** [rome/tools#4477] [Suggestions]
I already made a proof of concept, but currently fails with a obtuse error. Would be useful for someone who wanted to run the benchmarks themselves but possibly doesn't have the necessary depenencies installed (Rust)

https://github.com/connorjburton/rome/blob/benchmark-docker/Dockerfile

**Comment by ematipico** [comment]
That would be great, but we don't have the capacity to look after the benchmarks at the moment :(

We can support you if you'd like to lead the developments!

**Comment by connorjburton** [comment]
PR Raised https://github.com/rome/tools/pull/4494
